vercel.json

```diff
@@ -4,6 +4,11 @@
 			"destination": "https://opencollective.com/boston-ts-club",
 			"permanent": true,
 			"source": "/donate"
+		},
+		{
+			"destination": "https://forms.gle/HRDzvDoE2nWFDYTC6",
```

Comment:
Perfect, I wanted to add this, thank you.

This is itself a redirect to https://docs.google.com/forms/d/e/1FAIpQLSevxqyXjEE2AQ8C7xxkmoiiasr5ONIvPyAl8ad-MseNZ2ZBxA/viewform, I'm thinking we should use this directly, saves a round trip I guess. Thoughts?

Reply:
No reason to not do that. I can fix up rq now

Reply:
Fixed.